Mathlib/Data/Nat/ModEq.lean

```diff
@@ -141,6 +141,9 @@ protected theorem add_left_cancel (h₁ : a ≡ b [MOD n]) (h₂ : a + c ≡ b +
 protected theorem add_left_cancel' (c : ℕ) (h : c + a ≡ c + b [MOD n]) : a ≡ b [MOD n] :=
   ModEq.rfl.add_left_cancel h
 
+protected theorem add_left_cancel_iff (c : ℕ) : c + a ≡ c + b [MOD n] ↔ a ≡ b [MOD n] :=
+  ⟨ModEq.add_left_cancel rfl, ModEq.add_left _⟩
```

Comment:
Can you follow statement and naming of [`AddCommGroup.ModEq.add_iff_left`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/ModEq.html#AddCommGroup.ModEq.add_iff_left)?